pkg/configuration/configuration_test.go

```diff
@@ -495,3 +495,149 @@ func (s *TestConfigurationSuite) TestGetNamespace() {
 		assert.Equal(s.T(), newVal, config.GetNamespace())
 	})
 }
+
+func (s *TestConfigurationSuite) TestVerificationEnabled() {
+	key := configuration.EnvPrefix + "_" + "VERIFICATION_ENABLED"
+	resetFunc := UnsetEnvVarAndRestore(s.T(), key)
+	defer resetFunc()
+
+	s.Run("default", func() {
+		resetFunc := UnsetEnvVarAndRestore(s.T(), key)
+		defer resetFunc()
+		config := s.getDefaultConfiguration()
+		require.Equal(s.T(), true, config.GetVerificationEnabled())
```

Comment:
Or simply
```suggestion
		require.True(s.T(), config.GetVerificationEnabled())
```

Reply:
Of course :)  I've fixed this and the other instances.